lib/mrkt.rb

```diff
@@ -18,6 +19,7 @@ class Client
     include CrudLeads
     include CrudLists
     include ImportLeads
+    include CustomObjects
```

Comment:
Since it creates, reads, updates deletes the custom objects it should also called `CrudCustomObjects`.